linux_os/guide/system/permissions/files/permissions_within_important_dirs/root_permissions_syslibrary_files/rule.yml

```
@@ -59,3 +60,14 @@ ocil: |-
     To find if system-wide library files stored in these directories are not group-owned by
     root run the following command for each directory <i>DIR</i>:
     <pre>$ sudo find -L <i>DIR</i> ! -group root -type f </pre>
+
+template:
+    name: file_groupowner
+    vars:
+        filepath:
+            - /lib/
+            - /lib64/
+            - /usr/lib/
+            - /usr/lib64/
+        file_regex: ^.*$
+        filegid: '0'
```

```suggestion
        filegid: '0'
        recursive: 'true'
```

I believe we need to use the `recursive` option here. What do you think?

---

No need to as we have the file_regex, that is enough to search for every file under the specified filepath
The recursive flag is actually needed when you pass a filepath, that is a directory and there's no regex, so we make sure that we are searching for subdirectories as well